styles/services.scss

```diff
@@ -0,0 +1,4 @@
+@import "~@carbon/ibmdotcom-styles/scss/patterns/sections/leadspace/leadspace";
+@import "@carbon/ibmdotcom-styles/scss/patterns/blocks/content-block-segmented/index";
```

Comment:
add the `~` here since we can use webpack for building this.

Reply:
I have just added it.